.github/workflows/windows-ci.yml

```diff
@@ -0,0 +1,52 @@
+name: Windows CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  SBT_OPTS: '-Xmx6g -Xms1g -Xss4m'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        java: [ '8' ]
+
+    runs-on: windows-latest
```

Comment:
You are the windows guru: I guess we'll be less affected by ambient changes due to the lack of pinning then by breakages because of changing version support?

Reply:
That's the idea ;)